tests/tap.py

```diff
@@ -263,7 +263,7 @@ def _add_error(self, message):
     rules = Rules()
 
     try:
-        out = subprocess.check_output(sys.argv[1:], universal_newlines=True)
+        out = subprocess.check_output(' '.join(sys.argv[1:]), universal_newlines=True, shell=True)
```

Comment:
This change seems wrong and unrelated to the rest of this PR.

Reply:
Sorry, this was the change that I was trying to fix the problem, but doesn't work. I've removed it.